lesson5.js

```diff
@@ -0,0 +1,30 @@
+  const ul = document.getElementById('js-list');
+  const info = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ];
+  const fragment = document.createDocumentFragment();
+
+  function list() {
+    return new Promise((resolve) => {
+      resolve(info);
+    });
+  }
+  list().then(() => { 
```

Comment:
Now, let's try to receive the resolved value with `then()`.
Currently, we are getting the value directly from info.

 [This part](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then#return_value) of MDN is easy to understand.